docs/en/stack/ml/nlp/ml-nlp-deploy-models.asciidoc

```diff
@@ -134,10 +134,28 @@ When you deploy the model, it is allocated to all available {ml} nodes. The
 model is loaded into memory in a native process that encapsulates `libtorch`,
 which is the underlying machine learning library of PyTorch.
 
-TIP: You can set additional deployment options that affect the performance of
```

Comment:
Just above, there is a sentence:  ```When you deploy the model, it is allocated to all available {ml} nodes.```

That statement is no longer true. We need to change this to something like "When you deploy the model, its allocations are distributed across available {ml} nodes. etc.". But it feels like this sentence should go after we have explained what allocations are.